README.md

```diff
@@ -0,0 +1,38 @@
+What is this?
+=============
+
+The Map Activity uses a quick version of Google Maps for exploring the world. Mapmaking tools add photos, text, and Wikipedia articles to the map. You can also download maps and lessons from an online library.
+
+How to use?
+===========
+
+The Map Activity is not a part of Sugar desktop, but is often added.  Please refer to;
+
+* [How to Get Sugar on sugarlabs.org](https://sugarlabs.org/),
+* [How to use Sugar](https://help.sugarlabs.org/),
+
+How to upgrade?
+===============
+
+On Sugar desktop systems;
+* use [My Settings](https://help.sugarlabs.org/en/my_settings.html), [Software Update](https://help.sugarlabs.org/en/my_settings.html#software-update), or;
+* use Browse to open [activities.sugarlabs.org](https://activities.sugarlabs.org/), search for `Map`, then download.
+
+How to integrate?
+=================
+
+The Map Activity depends on Python, [Sugar Toolkit for GTK+ 3](https://github.com/sugarlabs/sugar-toolkit-gtk3), and GTK+ 3.
+
+The Map Activity is started by [Sugar](https://github.com/sugarlabs/sugar).
```

Comment:
This line gives an incomplete information to the reader. 
 - How to start from _Sugar_? 
 - Is _Sugar_ the only way to start the activity?
 - Can the activity be used stand alone?

Reply:
@pro-panda what changes do you suggest?
Thanks!